client/routes.js

```diff
@@ -4,9 +4,11 @@ import { Route, Switch } from 'react-router-dom';
 
 // components
 import App from './components/App';
-import Login from './pages/Login';
 import Protected from './components/common/Protected';
-import NotFound from './components/common/NotFound';
+import loadComponent from './utils/loadComponent'
```

Comment:
Since it's a component, let the filename and import variable adhere to that of the normal components so rename this file and line to be:

```import LoadComponent from './utils/LoadComponent';```

Also a semi-colon needs to be added here.